File: STRIVER/video/vid-5/pattern6/p6.cpp
//  1 2 3 4 
//  1 2 3
//  1 2 
//  1

#include <iostream>
using namespace std;

int main()
{
    int row = 5;

    for (int i = 1; i <= row; i++)
    {
        for (int j = 1; j <=  row-i; j++)
        {
            cout <<j<<" ";
        }
        cout << endl;
    }
}

File: STRIVER/video/vid-5/pattern6/p6.py
# 1 2 3 4 
# 1 2 3
# 1 2 
# 1

def p6(row:int)->None:
    for i in range(row):
        for j in range(1, row-i):
            print(j, end=" ")
        print("")

row = 5
p6(row)